cairo-contracts/packages/clients/src/cometbft/component.cairo

```diff
@@ -249,7 +249,7 @@ pub mod CometClientComponent {
             let subject_client_sequence = msg.subject_client_id.sequence;
             let substitute_client_sequence = msg.substitute_client_id.sequence;
 
-            let subject_client_state: CometClientState = self
+            let mut subject_client_state: CometClientState = self
```

Comment:
why is this `mut` needed?

Reply:
Oh forgot to remove it

Reply:
Fixed in bf0f1f963677e36c28a4f67cd39291e8375e9977